test/integration/basic_test.go

```diff
@@ -161,13 +173,14 @@ func FeatureContext(s *godog.Suite) {
 
 	s.Step(`Minishift (?:has|should have) state "([^"]*)"`, m.shouldHaveState)
 	s.Step(`Minishift should have a valid IP address`, m.shouldHaveAValidIPAddress)
-	s.Step(`executing "minishift ([^"]*)"`, m.executingCommand)
-	s.Step(`executing "oc ([^"]*)`, m.executingOcCommand)
+	s.Step(`execut[es|ing]+ "minishift ([^"]*)"`, m.executingCommand)
+	s.Step(`execut[es|ing]+ "oc ([^"]*)`, m.executingOcCommand)
 	s.Step(`([^"]*) should contain ([^"]*)`, m.commandReturnShouldContain)
 	s.Step(`([^"]*) should contain`, m.commandReturnShouldContainContent)
 	s.Step(`([^"]*) should equal ([^"]*)`, m.commandReturnShouldEqual)
 	s.Step(`([^"]*) should equal`, m.commandReturnShouldEqualContent)
 	s.Step(`([^"]*) should be empty`, m.commandReturnShouldBeEmpty)
+	s.Step(`([^"]*) should match /([^"]*)/`, m.commandReturnShouldMatch)
```

Comment:
It is mostly about the wording... This comments was given before doing the whole reviews was concluded.

the meaning of "match" and "equal" are too close that from the phrasing alone it is not easy to distinguish the actual use-case.